Apps/Playground/UWP/App.h

```diff
@@ -50,6 +51,9 @@ ref class App sealed : public Windows::ApplicationModel::Core::IFrameworkView
     Windows::Foundation::Collections::IVectorView<Windows::Storage::IStorageItem^>^ m_files;
     bool m_windowClosed;
     bool m_windowVisible;
+    bool m_leftPressed{false};
+    bool m_middlePressed{false};
+    bool m_rightPressed{false};
```

Comment:
nit: apps folder uses default VS conventions
```suggestion
    bool m_leftPressed{ false };
    bool m_middlePressed{ false };
    bool m_rightPressed{ false };
```

Reply:
Changed